modules/persistent-storage-snapshots-about.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-snapshots.adoc
+
+[id="persistent-storage-snapshots-about_{context}"]
+= About snapshots
+
+A volume snapshot is a snapshot taken from a storage volume in a cluster. The external snapshot controller and provisioner enable use of the feature in the {product-title} cluster and handle volume snapshots through the {product-title} API.
+
+With volume snapshots, a cluster administrator can:
+
+. Create a snapshot of a `PersistentVolume` bound to a `PersistentVolumeClaim`.
```

Comment:
I would make all of these items an unordered list instead of an ordered list. We're displaying the functionality that is available here, and the ordering is unimportant.